### ISSUE TITLE
Indicate command errors for snapctl via return codes

### ISSUE THREAD

**Issue by pchronz** [intelsdi-x/snap#1062]
When I call snapctl in the wrong way, the return code still is 0. The missing error code makes automation harder.

```
/usr/local/bin/snapctl load plugin /etc/snap/current/plugin/snap-collector-mock1
$?
```
returns 0, although it should be != 0, since load and plugin are swapped.

**Comment by mkhajeh** [comment]
That is resolved already in #1023, but it is not released yet

**Comment by pchronz** [comment]
@mkhajeh perfect, thanks. I'll try it out with the next release or on the master branch.
